clients/cobol-lsp-vscode-extension/src/services/LanguageClientService.ts

```diff
@@ -40,8 +40,12 @@ export class LanguageClientService {
   private handlers: Array<(languageClient: LanguageClient) => void> = [];
   private isNativeBuildEnabled: boolean = false;
   private executableService: NativeExecutableService;
+  private extensionUri: vscode.Uri;
 
-  constructor(private outputChannel: vscode.OutputChannel) {
+  constructor(
+    private outputChannel: vscode.OutputChannel,
+    storagePath: string,
```

Comment:
just
```suggestion
private storageUri: vscode.Uri,
```
and use it later in the relative pattern?

Reply:
should be resolved at : 8ce0f8515a9a2385de1d9abbff255fdab177093a